app/models/public_feed.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class PublicFeed < Feed
```

Comment:
The methods defined in the Feed weren't used at all, so it didn't seem necessary to inherit them.